fork-runner/src/main/java/com/shazam/fork/system/adb/Installer.java

```diff
@@ -50,7 +52,7 @@ public void run() {
                     logger.debug("Uninstalling {} from {}", appPackage, device.getSerialNumber());
                     device.uninstallPackage(appPackage);
                     logger.debug("Installing {} to {}", appPackage, device.getSerialNumber());
-					device.installPackage(appApk.getAbsolutePath(), true);
+					device.installPackage(appApk.getAbsolutePath(), true, GRANT_ALL_RUNTIME_PERMISSIONS);
```

Comment:
Unfortunately  in Lollipop and lower this flag might not work (```Error: Unknown option: -g```), We need an API level check 🙍